### DISCUSSION TITLE
Sidebar employmentType filter

### DISCUSSION THREAD

**Discussion by nickhaskell** [bullhorn/career-portal#520] [Q&A]
Is it possible to add an additional filter employmentType (eg. Contract, Direct-to-hire, Direct) filter to the sidebar? 

I've tried adding an additional <app-sidebar-filter> component to the sidebar.component.html file with what I assume would be the correct parameters, but this does not appear to be the case.

Is there any sort of documentation available for adding new filters?

```
    <app-sidebar-filter
        field="employmentType"
        title="{{ 'Employment Type' | translate }}"
        [filter]="filter"
        (checkboxFilter)="updateFilter('employmentType', $event)"
        data-automation-id="employmenttype"
    ></app-sidebar-filter>
```

**Comment by pbearne** [comment]
I can't help you with react.
But is give up type is standard sidebar in Matador Jobs https://matadorjobs.com/

**Reply by nickhaskell** [reply]
I'm sorry, what react are you referring to? The OSCP appears to be an angular and typescript app.

**Reply by pbearne** [reply]
My Bad, I can't help with JS ( angular, typescript )
